SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2371,7 +2404,7 @@ void USpatialNetDriver::ProcessPendingDormancy()
 			USpatialActorChannel* Channel = PendingDormantChannel.Get();
 			if (Channel->Actor != nullptr)
 			{
-				if (Receiver->IsPendingOpsOnChannel(*Channel))
+				if (Channel->HasPendingOps())
```

Comment:
nit: Think i'd be nicer if we inverted this check and let the ActorSystem do it, and internally it checks the ActorChannel.